backend/app/controllers/schedules_controller.ts

```diff
@@ -78,6 +78,18 @@ export default class SchedulesController {
       userId: userId,
     })
 
+    if (payload.groups !== undefined) {
+      await schedule.related('groups').sync(payload.groups.map((group) => group.id))
+    }
+
+    if (payload.registrations !== undefined) {
+      await schedule.related('registrations').sync(payload.registrations.map((group) => group.id))
+    }
+
+    if (payload.courses !== undefined) {
+      await schedule.related('courses').sync(payload.courses.map((group) => group.id))
+    }
+
```

Comment:
Czemu już tutaj dodajemy groupsy, rejegistrations i courses do planu? Zamysł miałem taki, zeby po kliknięciu dodania planu na froncie szedł request po prostu z domyślną nazwą. Czemu trzeba było to zmienić?